[ONNX] Fix pad Circular Mode

titaiwangms opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #86984

In https://github.com/pytorch/pytorch/pull/73433, a ONNX test case is missed, and the result is incorrect when it is converted to ONNX.

titaiwangms committed:
[ONNX] Fix pad Circular Mode

[ghstack-poisoned]

AllenTiTaiWang commented on torch/onnx/symbolic_opset9.py:
```diff
@@ -1849,32 +1850,34 @@ def constant_pad_nd(g: jit_utils.GraphContext, input, padding, value):
     )
 
 
-@_onnx_symbolic("aten::_pad_circular")
```
I deleted this, as this function is actually a subfunction of "aten::pad"

AllenTiTaiWang commented:
@BowenBao PTAL

BowenBao reviewed (approved):
🚀

AllenTiTaiWang commented:
@pytorchbot rebase

titaiwangms committed:
Update on "[ONNX] Fix pad Circular Mode"


In https://github.com/pytorch/pytorch/pull/73433, a ONNX test case is missed, and the result is incorrect when it is converted to ONNX.

[ghstack-poisoned]

AllenTiTaiWang commented:
@pytorchbot merge -g
